### DISCUSSION TITLE
List of values for an option

### DISCUSSION THREAD

**Discussion by geekscrapy** [saulpw/visidata#1167] [Ideas]
This came up, but I wondered if it was actually required.
It would be nice to be able to specify multiple URLs for plugins. I would use this to have plugins installable from the official repo and an internal one. However at the moment, it's not possible to edit a list of options within the options sheet.

A couple of questions surrounding this

1. Do people have internal and external plugin repos?
2. Are there any other options where it makes sense to have multiple values? motd is the only one I can really think of, and that's not functionally required

**Comment by geekscrapy** [comment]
An idea to assist with entering multiple values in an option might be to allow input of JSON string, for example, could be a list?

**Comment by saulpw** [comment]
I think you're right, @geekscrapy, there may not be much use for multiple options like this.  We may want to use a list or dict as the "default" for some options, which would specify an enumeration of possible option values.  But otherwise one value per option, I think.
